neighbour-model/src/test/java/no/vegvesen/ixn/federation/model/NeighbourTest.java

```diff
@@ -118,33 +118,33 @@ public void shouldCheckSubscriptionRequestsForUpdates_neighbourWithNewCapability
 		Neighbour neighbourWithNewerCapabilitiesThanSubscriptionRequest = new Neighbour("nice-neighbour", capabilities, null, fedIn);
 		assertThat(neighbourWithNewerCapabilitiesThanSubscriptionRequest.shouldCheckSubscriptionRequestsForUpdates(Optional.of(localSubscriptionUpdateTimeBeforeCapabilityPost))).isTrue();
 	}
-
+	//Flytte til en egen test-klasse som tester ConnectionBackoff
```

Comment:
REmove this test if it's already in ConnectionBackupTest

Reply:
Done :)